DatabaseTemplate/Modules/get-template-from-web.ps1

```diff
@@ -73,7 +73,7 @@ if ($isArchiveFile) {
     else {
         EnsureCommandIsAvailable "7z"
         $arguments = @("x", $archiveBackupFilePath, "-o$global:templateDatabaseFolder")
-        7z @arguments
+        7z @arguments | Out-Null
```

Comment:
This script doesn't belong to the current group, but while testing this group I found that 7z output was being returned to the caller, so I forwarded its output to null to be consistent with the behavior in Windows.